db/institutions.yml

```diff
@@ -584,6 +606,29 @@ lehighu:
   name: Lehigh University
   state: PA
   zip: '18015'
+m.p.instituteofsocialscienceresearch:
+  aka:
+    - MPISSR
+  city: Ujjain
+  country: India
+  day: 19
+  month: 2
+  name: M.P. Institute of Social Science Research
```

Comment:
This should be max planck institution for solid state research, or sthg like that.

Reply:
@sbillinge this one is called "M.P. Institute of Social Science Research, Ujjain" on its official website corresponding to "mpissr" acronym. "Max Planck Institute for Chemical Physics of Solids" is "mpicps" that is in Germany.